src/foam/nanos/ruler/Ruled.js

```diff
@@ -39,11 +43,16 @@
       writePermissionRequired: true
     },
     {
-      class: 'foam.mlang.predicate.PredicateProperty',
+      class: 'FObjectProperty',
+      of: 'foam.mlang.predicate.Predicate',
       name: 'predicate',
       documentation: 'predicate is checked against an object; if returns true, the action is executed. Defaults to return true.',
-      factory: function () { return this.TRUE; },
-      javaFactory: 'return foam.mlang.MLang.TRUE;',
+      factory: function () { 
+        return this.True.create();
```

Comment:
what is the different between `this.True.create()` and `this.TRUE`? why the latter no longer works?

Reply:
@jinn9 adding this specifically to make things work in the UI. I'm just synching Ruled and Rule

Reply:
isn't one for client and the other for backend?

Reply:
This is how TRUE is defined in mlang, so I believe the two are equivalent.

{
      name: 'TRUE',
      factory: function() { return foam.mlang.predicate.True.create() }
 },